src/map.tsx

```diff
@@ -139,24 +124,25 @@ const ReactMapboxFactory = ({
   bearingSnap = 7,
   antialias = false,
   mapInstance,
-  transformRequest
+  transformRequest,
+  region = 'ww',
```

Comment:
```suggestion
  region = TrimbleMaps.Common.Region.WW,
```
⛏️ I think instead of using this magical constant it would be prefable to use the constant provided by the library so if in the future it changes during SDK upgrade then we'll get an error.

Reply:
Good find! I couldn't find an enum value myself before, which is why I used the constant.